yt/utilities/sdf.py

```diff
@@ -414,7 +414,7 @@ def parse_struct(self, line, ascfile):
                 str_types.append((v, vtype))
             l = ascfile.readline()
         num = l.strip("}[]")
-        num = num.strip("\;\\\n]")  # NOQA B005
+        num = num.strip("\\;\\\n]")  # NOQA B005
```

Comment:
so far this is the only change that raises my eyebrow. This line is supposed to remove every occurence of `\`, `;` and `\n` from the `num`.
1) I probably wrote the NOQA comment myself, but I think it ought to be removed
2) I'm not sure why or how pyupgrade decided to edit this
3) This is also removing `]` occurrences, though this is redundant with the previous line so this part is a null op.